scripts/globals/mobskills.lua

```diff
@@ -271,6 +271,10 @@ xi.mobskills.mobPhysicalMove = function(mob, target, skill, numberofhits, accmod
         skill:setMsg(xi.msg.basic.SKILL_MISS)
     end
 
+    if target:getMod(xi.mod.PET_DMG_TAKEN_PHYSICAL) ~= 0 then
```

Comment:
should we do a nil check to see if they have the mod at all? since we possibly could want to make pets do 0 dmg against certain things?

Reply:
Big true, forgot it could be nil.

Reply:
So it actually can't be nil, because mods always default to 0, so the lowest someone can theoretically go is 1%.